### PR TITLE
MTV-789: Release Notes for MTV 2.4.3

### PR DESCRIPTION
**Pull request by anarnold97**
### Jira

* [MTV-789](https://issues.redhat.com/browse/MTV-789): Release notes for MTV 2.4.3

### CODE REVIEW

**Comment by anarnold97** [comment]
@ahadas && @qiyuann - please see the first draft for 2.4.3
Thanks

File: documentation/modules/rn-2.4.adoc [hunk 1]
```diff
@@ -135,6 +135,47 @@ When upgrading from MTV 2.4.0 to a later version, the operation fails with an er
...
 This release has the following resolved issues:
 
+.Multiple HTTP/2 enabled web servers are vulnerable to a DDoS attack (Rapid Reset Attack)
+
+A flaw was found in handling multiplexed streams in the HTTP/2 protocol. In previous releases of MTV, the HTTP/2 protocol allowed a denial of service (server resource consumption) because request cancellation could reset multiple streams quickly. The server had to set up and tear down the streams while not hitting any server-side limit for the maximum number of active streams per connection, which resulted in a denial of service due to server resource consumption.
+
+This issue has been resolved in MTV 2.4.3 and 2.5.2. It is advised to update to one of these versions of MTV or later.
+
+// as the following Jira tickets are all set to "Security Level:Red Hat Employee (Red Hat Employee and Contractors only)"
+// unless the security level is changed, i would probably remove but will defer.
```
Comment (ahadas) [inline documentation/modules/rn-2.4.adoc:145]: yeah, we should remove those

File: documentation/modules/rn-2.4.adoc [hunk 2]
```diff
@@ -135,6 +135,47 @@ When upgrading from MTV 2.4.0 to a later version, the operation fails with an er
...
+The following issues have been listed under this issue:
+
+* link:https://issues.redhat.com/browse/MTV-750[CVE-2023-44487 mtv-console-plugin-container: HTTP/2: Multiple HTTP/2 enabled web servers are vulnerable to a DDoS attack (MTV-750)]
+* link:https://issues.redhat.com/browse/MTV-758[CVE-2023-39325 CVE-2023-44487 mtv-api-container: various flaws (MTV-758)]
+* link:https://issues.redhat.com/browse/MTV-762[CVE-2023-39325 mtv-must-gather-api-container: golang: net/http, x/net/http2: rapid stream resets can cause excessive work (MTV-762)]
+* link:https://issues.redhat.com/browse/MTV-764[mtv-validation-container: golang: net/http, x/net/http2: rapid stream resets can cause excessive work (MTV-764)]
+
+For more information, see link:https://access.redhat.com/security/cve/cve-2023-44487[CVE-2023-44487 (Rapid Reset Attack)] and link:https://access.redhat.com/security/cve/cve-2023-39325[CVE-2023-39325 (Rapid Reset Attack)].
+
+.Gin Web Framework does not properly sanitize filename parameter of Context.FileAttachment function
```
Comment (ahadas) [inline documentation/modules/rn-2.4.adoc:155]: as that's the release notes for 2.4 - this section should be dropped (only CVE-2023-44487 and CVE-2023-39325 are handled)

File: documentation/modules/rn-2.4.adoc [hunk 3]
```diff
@@ -135,6 +135,47 @@ When upgrading from MTV 2.4.0 to a later version, the operation fails with an er
...
+
+* link:https://issues.redhat.com/browse/MTV-600[CVE-2023-29401 mtv-controller-container: golang-github-gin-gonic-gin: Gin Web Framework does not properly sanitize filename parameter of Context.FileAttachment function]
+* link:https://issues.redhat.com/browse/MTV-601[CVE-2023-29401 mtv-must-gather-api-container: golang-github-gin-gonic-gin: Gin Web Framework does not properly sanitize filename parameter of Context.FileAttachment function]
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=2203776[CVE-2023-26125 mtv-controller-container: golang-github-gin-gonic-gin: Improper Input Validation]
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=2203777[ CVE-2023-26125 mtv-must-gather-api-container: golang-github-gin-gonic-gin: Improper Input Validation]
+
+For more information, see link:https://access.redhat.com/security/cve/cve-2023-44487[CVE-2023-44487 (Rapid Reset Attack)] and link:https://access.redhat.com/security/cve/CVE-2023-26125[CVE-2023-26125]
+
+
+.CVE-2023-26144 mtv-console-plugin-container: graphql: Insufficient checks in the OverlappingFieldsCanBeMergedRule.ts
```
Comment (ahadas) [inline documentation/modules/rn-2.4.adoc:171]: as that's the release notes for 2.4 - this section should be dropped (only CVE-2023-44487 and CVE-2023-39325 are handled)

Comment (anarnold97) [reply documentation/modules/rn-2.4.adoc:164]: @ahadas - please can i clarify _

* Remove Gin Web Framework CVE
&&
* Remove GraphQL CVE

Comment (ahadas) [reply documentation/modules/rn-2.4.adoc:164]: correct, only `.Multiple HTTP/2 enabled web servers are vulnerable to a DDoS attack (Rapid Reset Attack)` is relevant for 2.4.3

**Review by ahadas — APPROVED** [review]

**Pull request merged**